lib/reactotron-react-native/src/reactotron-react-native.ts

```diff
@@ -65,11 +65,29 @@ const DEFAULTS: ClientOptions<ReactotronReactNative> = {
     ...getReactNativeDimensions(),
   },
   /* eslint-disable @typescript-eslint/no-use-before-define */
-  getClientId: async () => {
+  getClientId: async (name: string = "") => {
     if (reactotron.asyncStorageHandler) {
       return reactotron.asyncStorageHandler.getItem(REACTOTRON_ASYNC_CLIENT_ID)
     }
 
+    // Generate clientId based on the device info
+    const { screenWidth, screenHeight, screenScale } = getReactNativeDimensions()
+
+    // Accounting for screen rotation
+    const dimensions = [screenWidth, screenHeight].sort().join("-")
```

Comment:
u fancy

Reply:
Yeah... overkill really. I originally had the scale in there too, but didn't like the way that it read. 😅

Might be better to do '${smaller}x${larger}' with a ternary.